plugin/src/main/groovy/com/novoda/gradle/command/VariantSuffix.groovy

```diff
@@ -0,0 +1,11 @@
+package com.novoda.gradle.command
+
+class VariantSuffix {
+
+  static String variantNameFor(variant) {
+    def buildTypeName = variant.buildType.name.capitalize()
+    def projectFlavorNames = variant.productFlavors.collect { it.name.capitalize() }
+    def projectFlavorName = projectFlavorNames.join()
```

Comment:
This one can be joined with the previous line, imho.

Reply:
:+1: I believe this could also be just `variant.name.capitalize()`
`variant.name` does this concatenation for us. Will verify and do a follow-up on this.